### PR TITLE
Added Badge button components

### PR DESCRIPTION
**Pull request by rutu14**

### CODE REVIEW

File: badge.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+        </div>
+    </div>
+    <script src="script.js" ></script>
+</body>
+</html>
+<!-- style="width: 809px; height: 356px; border:0; transform: scale(1); overflow:hidden;" -->
```
Comment (iRohitGaur) [inline badge.html]: remove this commented code?

File: components/badge.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    height:1.4rem;
+    width: 1.4rem;
+    background-color: var(--info-text);
+    transform: translate(160%, 100%);
+}
+
+.badge-xl{
+    height:2rem;
+    width: 2rem;
+    border-width: 0.3rem;
```
Comment (iRohitGaur) [inline components/badge.css]: you can give px value to anything below 10px

File: components/button.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+.btn-primary{
+    cursor: pointer;
+    border: none;
+    background-color: var(--primary-colour);
+    color: var(--tertiary-colour);
+    font-size: 1.4rem;
+    line-height: 3.6rem;
+    font-weight: 500;
+    letter-spacing: .0892857143em;
```
Comment (iRohitGaur) [inline components/button.css]: why .0892857143em?

**Review by iRohitGaur — APPROVED** [review]

**Comment by archit7448** [comment]
good to merge

**Review by archit7448 — APPROVED** [review]

File: components/button.css (hunk 3 above)
Comment (rutu14) [reply components/button.css]: Spelling error

**Pull request merged**